compiling/extract_and_compile.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+tar xzf NthPrime.tgz
+cd NthPrime
```

Comment:
You should look at the warning that `shellcheck` gives you about this line, and definitely ask questions about it if it's not clear what they're worried about.